docs/source/beginner-tutorial.rst

```diff
@@ -140,8 +140,8 @@ on OSX):
 
 As you can see there are a couple of flags that specify where to find your
 algorithm (``-f``) as well as parameters specifying which data to use,
-defaulting to the :ref:`quandl-data-bundle`. There are also arguments for the
-date range to run the algorithm over (``--start`` and ``--end``). Finally,
+defaulting to the :ref:`quantopian-quandl-mirror`. There are also arguments for
```

Comment:
a ref link without an explicit link name will use the name of the next section. this renders as 'Quantopian Quandl WIKI Mirror': https://github.com/quantopian/zipline/pull/1178/files/265f77bf3e0ea53535fa6f025d44f5d05e0b2f48#diff-d2e46d2591b3fdb2539b3947867f60a5R106